CHANGELOG.asciidoc

```diff
@@ -6,6 +6,96 @@
 ////////////////////////////////////////////////////////////
 // Template, add newest changes here
 
+=== Beats version avoid-hardcoded-slash-log-in-docker-path
+https://github.com/jp-paillet/beats.git[Check the HEAD diff]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+*Auditbeat*
+
+*Filebeat*
+
+- Allow beats to parse docker inputs outside an hardcoded directory structure. {pull}9620[9620]
+
+*Heartbeat*
+
+*Journalbeat*
+
+*Metricbeat*
+
+*Packetbeat*
+
+*Winlogbeat*
+
+*Functionbeat*
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+*Auditbeat*
+
+*Filebeat*
+
+- Reworked the initialization of container's paths. {pull}9620[9620]
+
+*Heartbeat*
+
+*Journalbeat*
+
+*Metricbeat*
+
+*Packetbeat*
+
+*Winlogbeat*
+
+*Functionbeat*
+
+==== Added
+
+*Affecting all Beats*
+
+*Auditbeat*
+
+*Filebeat*
+
+- Added `containers.suffix` input configuration option to avoid the static addition of the "/*.log" in paths. {issue}8988[8988] {pull}9620[9620]
```

Comment:
You can just add this under `Beats version HEAD` into the right section. No need to introduce all this content. Should be in the end only be 1 line.

Reply:
Hi Nicolas, thanks for your clarifications, but I'm now lost into git actions... I'll try to end this by today anyway.

Reply:
Hi Nicolas, I wish my last changes will feet your expectations. I'll be on holidays for the next 2 weeks to come, so I'll look at this in January. Again, thanks for your guidance. Merry Christmas.

Reply:
@jp-paillet Thanks for pushing this forward. We will probably not get to review this one today but lets try to get this in beginning next year. Merry Christmas, happy new year 🎉